src/parser/rules.rs

```diff
@@ -159,7 +159,13 @@ impl Parser {
             _ => None,
         };
 
-        let body = self.parse_block()?;
+        let peeked_kind = self.peek()?.kind; 
+        let body = if peeked_kind == TokenKind::CurlyBracesOpen {
+            self.parse_block()?
+        } else {
+            self.next()?;
+            self.parse_statement()?
```

Comment:
We should somehow make clear that this is an inline function declaration. Maybe by wrapping it inside a `parse_inline_function` of sorts?

Reply:
Got it and I should consume the equal sign inside that function.